localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -587,6 +587,9 @@
         <trans-unit id="columnWidthMustBePositiveError">
             <source xml:lang="en">Width cannot be 0 or negative</source>
         </trans-unit>
+        <trans-unit id="objectExplorerNodeRefreshError">
+            <source xml:lang="en">Error occured while refreshing nodes</source>
```

Comment:
Error messages displayed to the user should be as helpful as possible - this is too generic to be much use. Something like `An error occurred refreshing the {NodeName} node. See the [MSSQL output](command:<command here>)` would be much more useful IMO. 

Although I suppose we generally already have logic to display an error in the tree view itself if node expansion fails - so it would probably be worth it to check if that happens when refreshing nodes as well (in which case displaying a separate message to the user is probably unnecessary)